### PR TITLE
[libuv] Update to 1.45.0

### PR DESCRIPTION
**Pull request by ralpioxxcs**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

---
I accidentally closed the PR, open PR again (previous: https://github.com/microsoft/vcpkg/pull/32072)
* Update libuv version to **1.45.0** from **1.44.2**
* Update all consuming ports using libuv
  * Change patch files using old target name (`uv`, `uv_a` -> `libuv::uv` , `libuv::uv_a`)

### CODE REVIEW

**Comment by JonLiu1993** [comment]
@ralpioxxcs, Thanks for your PR, please fix the conflicting files.

**Comment by JonLiu1993** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

**Review by JonLiu1993 — DISMISSED** [review]

**Comment by JonLiu1993** [comment]
Tested usage successfully by libuv:x64-windows:
````
libuv provides CMake targets:

    find_package(libuv CONFIG REQUIRED)
    target_link_libraries(main PRIVATE $<IF:$<TARGET_EXISTS:libuv::uv_a>,libuv::uv_a,libuv::uv>)
````

File: versions/u-/usockets.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "648715b141f4bb501ebc83d09da547ee0b3fe467",
+      "git-tree": "91880c7699f94234d2610996007f9f096e2b786b",
       "version": "0.8.6",
       "port-version": 0
```
Comment (vicroms) [inline versions/u-/usockets.json:6]: Need to increase `port-version` on `usockets` and run `x-add-version` again.

**Review by vicroms — CHANGES_REQUESTED** [review]

File: versions/u-/usockets.json (hunk 1 above)
Comment (ralpioxxcs) [reply versions/u-/usockets.json:6]: > Need to increase `port-version` on `usockets` and run `x-add-version` again.

I have tried `vcpkg x-add-version usockets` but nothing changed in my forked repository.

```text
version 0.8.6 is already in /home/edge/workspace/vcpkg_ralpioxxcs/versions/u-/usockets.json
version 0.8.6 is already in /home/edge/workspace/vcpkg_ralpioxxcs/versions/baseline.json
No files were updated for usockets
```
is it required to sync master?

Comment (vicroms) [reply versions/u-/usockets.json:6]: Did you commit your changes to `ports/usockets/vcpkg.json`? 
It is always advisable to merge with latest.

Comment (JonLiu1993) [reply versions/u-/usockets.json]: @ralpioxxcs, I have already modified it for you. This problem has been solved. The problem is that you did not add `"port-version": 1` in `vcpkg.json` when you modified usockets, thanks for the reminder of @vicroms.

Comment (ralpioxxcs) [reply versions/u-/usockets.json]: I realized that I forgot to update the port version after resolving conflicts. Thanks! @JonLiu1993

**Review by vicroms — APPROVED** [review]

**Comment by autoantwort** [comment]
> Update all consuming ports using libuv
> 
>     * Change patch files using old target name (`uv`, `uv_a` -> `libuv::uv` , `libuv::uv_a`)

Seems that https://github.com/microsoft/vcpkg/issues/32328 was missed :)

**Pull request merged**